Feature/num 505 integration test diabete mellitus

af-dpe opened the pull request:
Added 4 json's for diabetes mellitus 
added create keyword for diabetes mellitus 
added 4 test cases for diabetes mellitus: 
- default 
- Type 1 
- Type 2 
- Type 2 Insulin treaded

ppazos commented:
@BirgitSaalfeld can you please check the conflicts with develop branch?

BirgitSaalfeld commented:
Dear @ppazos, I reintegrated the current develop branch.

ppazos commented on src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:
```diff
@@ -278,15 +278,17 @@ public void createObservationUsingDefaultProfile() {
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Default profile is not supported for Observation. One of the following profiles is expected: " +
```
@BirgitSaalfeld please fix the indentation to 4 spaces.

BirgitSaalfeld replied:
adapted indentation

ppazos commented on src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:
```diff
@@ -299,15 +301,18 @@ public void createObservationUsingUnsupportedProfile() {
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. " +
```
@BirgitSaalfeld please fix the indentation to 4 spaces.

BirgitSaalfeld replied:
Four spaces counted from what? I learned programming guidlines like this. Do we have guidlines in this project or is there an auto-formatting we can use?

ppazos replied:
Hi @BirgitSaalfeld please check the 'green' vs. 'red', the red code is 4 spaces to the right from the container line 'Assertions.assertEquals(' while the green is 8 spaces to the right from that.

ppazos replied:
BTW for the format I think we use the SonarLint plugin

- https://www.sonarlint.org/vscode
- https://www.sonarlint.org/intellij

BirgitSaalfeld replied:
done

BirgitSaalfeld commented:
same queston as in the other pull request.

Commit:
Added 4 json's for diabetes mellitus
added create keyword for diabetes mellitus
added 4 test cases for diabetes mellitus:
* default
* Type 1
* Type 2
* Type 2 Insulin treaded

Commit:
fix for keyword arguments

wlad committed:
adds tag and fixes typo [skip ci]

wlad reviewed (approved):
looks good 👍
